src/builtins/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I needed to specify the context used by files in `builtins`, as a browser context. Some files are using features specific to browsers (such as `WebSocket` for what I remember)